pkg/user/tx_client.go

```diff
@@ -138,6 +142,44 @@ func WithAdditionalCoreEndpoints(conns []*grpc.ClientConn) Option {
 	}
 }
 
+// WithTxWorkers enables parallel transaction submission with the specified number of worker accounts.
+// Worker accounts are automatically generated with hardcoded names unless numWorkers is 1, in which case
+// the existing default account is used.
+// By default, workers are initialized automatically when SetupTxClient is called.
+func WithTxWorkers(numWorkers int) Option {
+	if numWorkers <= 0 {
+		return func(*TxClient) {}
+	}
+
+	return func(c *TxClient) {
+		pool := newParallelTxPool(c, numWorkers, true)
+		c.parallelPool = pool
+	}
+}
+
+// WithTxWorkersNoInit enables parallel transaction submission without automatic initialization.
+// InitializeWorkerAccounts must be called manually when using this option.
+func WithTxWorkersNoInit(numWorkers int) Option {
```

Comment:
also forgot to mention that we can initialize automatically or manually later using the method